tcl/dialog_iemgui.tcl

```diff
@@ -12,6 +12,24 @@ namespace eval ::dialog_iemgui:: {
     namespace export pdtk_iemgui_dialog
 }
 
+source [file join [file dirname [info script]] "dialog_iemgui_knob.tcl"]
+
+proc procExists p {
+    return uplevel 1 [expr {[llength [info procs $p]] > 0}]
+}
+
+proc ::dialog_iemgui::callGUIproc {mytoplevel p args} {
```

Comment:
This could also be snake case.

Reply:
...and `::dialog_iemgui::call_gui_proc`
I'm gonna change these.

Reply:
done